test/cell_test.rb

```diff
@@ -59,5 +59,17 @@ def test_fire_upon_lowers_ship_health_by_one
     @cell.place_ship(@submarine)
     @cell.fire_upon
     assert_equal 1, @cell.ship.health
+    assert_equal 2, @cell.ship.health
+  end
+
+  def test_cell_has_not_been_fired_upon
+    assert_equal false, @cell.fired_upon?
+    @cell.fire_upon
+    assert_equal true, @cell.fired_upon?
+  end
+
+  def test_fire_upon_reduces_health
+    @cell.fire_upon
+    assert_equal @ship.health, @ship.length - 1
```

Comment:
Yep, this works! We could also hard-code the expected value of @ship.health, like this:
`assert_equal 2, @ship.health`

Reply:
We might want to add some more code and assertions after this to test whether it works on a different cell (@cell_2) and a different ship (submarine), like we did with the earlier ones.